src/els_parser.erl

```diff
@@ -285,10 +285,11 @@ macro(Tree) ->
 -spec record_access(tree()) -> [poi()].
 record_access(Tree) ->
   RecordNode = erl_syntax:record_access_type(Tree),
-  case erl_syntax:type(RecordNode) of
-    atom ->
+  FieldNode = erl_syntax:record_access_field(Tree),
+  case {erl_syntax:type(RecordNode), erl_syntax:type(FieldNode)}  of
```

Comment:
I thought we wanted to support cases when a macro is used instead of an atom.
If I'm reading this correctly, we are explicitly ignoring all cases where the field name is an atom.
This could be problematic if, for example, in the future we want to find all accesses to a record field.
Isn't it possible to add support for macros instead?

Reply:
> If I'm reading this correctly, we are explicitly ignoring all cases where the field name is an atom.

Did you mean we  are ignoring all cases where the field name is a *macro*?

> Isn't it possible to add support for macros instead?

It is possible but I don't know what you have in mind though? Are you proposing to:
1. Keep track of the values for all macros defined in the source (`.hrl` files included) and try to figure out the value of the macro used in the field?
2. Or register the record access with a dummy value for the field?

I would vote for option 2 since the option 1 get complicated very fast (i.e. what happens with macros with arguments?).

Reply:
I would also lean towards `2`.